README.md

```diff
@@ -1,3 +1,7 @@
+<p align="center">
+  <img src="/assets/images/logo/stark_logo_black.svg" alt="Stark : a powerful front-end framework" width="226" height="50">
```

Comment:
Good news and bad new for this: The logo is shown but... the size is wrong and the font is not the same as in the showcase :s.. 

![logo on readme](https://user-images.githubusercontent.com/5698318/47206164-3996d780-d388-11e8-82aa-28fe6935a261.PNG)

I guess the issue about the fonts is because the browser doen't know about the font embedded in the SVG... so I think we should adapt the `font-family` of the SVG to use a fallback, which is the same that we also define as`font-family` in the SCSS of Stark.